client/components/vertical-nav/index.jsx

```diff
@@ -2,15 +2,17 @@
  * External dependencies
  */
 import React from 'react';
+import style from './style';
+import withStyles from 'isomorphic-style-loader/lib/withStyles';
 
 const VerticalNav = React.createClass( {
 	render() {
 		return (
-			<div className="vertical-nav">
+			<div className={ style.verticalNav }>
```

Comment:
How is `verticalNav` defined? Is there meant to be a `style.scss` file included in this branch? Curious how this looks from a stylesheet writing point-of-view.